src/classes/HH_OppContactRoles_TDTM.cls

```diff
@@ -65,28 +77,28 @@ public class HH_OppContactRoles_TDTM extends TDTM_Runnable {
                // UNDONE: waiting to add the actual custom setting to use.
         	if (strRefreshSetting == null || strRefreshSetting == 'none')
         	   return null;
-        	boolean isOpenOnly = (strRefreshSetting == 'Open Opportunities');
+        	Boolean isOpenOnly = (strRefreshSetting == 'Open Opportunities');
```

Comment:
Just a comment for when we refactor OCR's we should strip this code, I do not believe this feature was fully implemented.

Reply:
`isOpenOnly` is used on line 84. Also, there is a unit test (`updateOppHHAccountRefreshOpen()`](https://github.com/SalesforceFoundation/Cumulus/blob/master/src/classes/HH_OppContactRoles_TEST.cls#L386) testing `Open Opportunities`. If this functionality is not used, I think we should create another WI to address removal of it. Thoughts?

Reply:
I have such a terrible memory, ... strRefreshSetting  is being checked for null or none and then the boolean is the result of comparing to open opps here on 80 but the variable does not vary, it is set on line 38 and then never touched again.

Reply:
Created W-031780.